src/pyscaffold/templates/sphinx_conf.template

```diff
@@ -99,11 +93,12 @@ copyright = "${year}, ${author}"
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
+# The following are default values, that are rewritten later in the script.
 #
 # The short X.Y version.
-version = ""  # Is set by calling `setup.py docs`
+version = os.getenv("READTHEDOCS_VERSION", "")
```

Comment:
This env var contains the same kind of string as the ones included in the RTD version switch menu, e.g. `v.4.0.x`, `stable`, `latest`.